content/A take on empowering ActiveRecord.md

```diff
@@ -0,0 +1,63 @@
+# A take on empowering ActiveRecord
+
+I think we are quite underpowered when it comes to doing queries on Rails, having this in mind I proposed my opinion on how we could improve this and made a PR. I also wanted to share it here to know what others think and maybe get some traction.
+
+## Querying with Ruby blocks
+
+My idea of how querying can be enhanced is simple: use a block to build the query just as if you were using `#select` or `#map`.  Here's a quick example on how I imagine that syntax looking:
+
+```ruby
+Post.where { |post| post.created_at.gt(1.month.ago) }
+```
+
+Wouldn't you agree that's much better writing/parsing SQL strings? You may also notice that I purposely used Arel's syntax when comparing values, that's because I think Arel is great and I find myself using it a lot in complex queries. Although I think that we could also just alias the `gt`, `gteq`, `lteq` methods and so on and have something like this too:
+
+```ruby
+Post.where { |post| post.created_at > 1.month.ago }
+```
+
+I mean other ORMs have had this power for years now, here's an example straight from [Sequel](https://github.com/jeremyevans/sequel)'s documentation:
+
+```ruby
+Post.where{ num_comments < 7 }
+```
+
+They describe the block as "magical" but I don't think there's anything magic going on, we are pretty used to using blocks everywhere, [FactoryBot](https://github.com/thoughtbot/factory_bot) and [Rspec](https://rspec.info/) are the first examples that come to my mind, where they've built a great DSL with blocks and the use of `instance_exec`.
```

Comment:
who is `They` ? I'd rather say `Some might find using blocks as "magical"...`

Reply:
I meant the Sequel people, I'll clarify